lib/screens/forum/screen.dart

```diff
@@ -157,30 +160,19 @@ class _ForumScreenState extends State<ForumScreen> with Screen {
                               splashColor: Colors.transparent,
                             ),
                           ),
-                          // IconButton(
-                          //   icon: Icon(Icons.search),
-                          //   color: Colors.pinkAccent[400],
-                          //   highlightColor: Colors.transparent,
-                          //   splashColor: Colors.transparent,
-                          //   onPressed: () {
-                          //     showSearch(
-                          //       context: context,
-                          //       delegate: ForumSearch(),
-                          //     );
-                          //   },
-                          // ),
                         ],
                       ),
                     ),
                     Expanded(
-                      child: StreamBuilder<List<PostCard>>(
+                      child: StreamBuilder(
```

Comment:
StreamBuilder<List<PostCard>> => StreamBuilder; parameterization provides stricter typing that might be useful for debugging

Reply:
i changed the asyncsnapshot to contain the specific type of the stream instead for consistency